src/scripts/components/showcase-selector/showcase-selector.tsx

```diff
@@ -32,7 +32,7 @@ const ShowcaseSelector: FunctionComponent = () => {
         backLink="/"
         backButtonId="backToDataMode"
         title={intl.formatMessage({id: 'showcaseMode'})}>
-        <div className={styles.playButton}>
+        <React.Fragment>
```

Comment:
Do we need this here and why? if we really need a div for styling issues then it should be a real div and not a fragment.